section3/reflection.md

```diff
@@ -1,17 +1,27 @@
+Maybe Pom will work better for me if i can get used to it and can start guessing my workflow better. But I really hate timers... I've never been bad at taking breaks though - at the same  time, maybe that's why i feel like i have trouble with discipline. I will give Pom another chance.
 
 2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
+i found pomodoro to be a little distracting and actually took away my focus. i like pausing at appropriate intervals as they align with the natural flow of my thought process. i would prefer to keep working past 20-30 minutes if it means staying on a train of thought or flow. interruptions kind of inhibit me.  i would rather set a longer timer, or make myself get up either when i come to the end of a Part/Section, when i find my mind wandering, when i'm getting frustrated/stuck past 25 minutes, or set longer intervals. Pom might work better for me once i get a better feel of my coding workflow, or if i can find an alert that doesn't interrupt my train of thought.
 
 3. What is a Hash, and how is it different from an Array?
+hashes store keys/symbols and values to their indexes, whereas arrays' elements are simpler values stored to indexes. hashes allow for storing more information
 
 4. In the space below, create a Hash stored to a variable named `pet_store`.  This hash should hold an inventory of items and the number of that item that you might find at a pet store.
+pet_store = {hamsters: 15, hamster_bedding_lbs: 150, hamster_habitrails: 8}
 
 5. Given the following `states = {"CO" => "Colorado", "IA" => "Iowa", "OK" => "Oklahoma"}`, how would you access the value `"Iowa"`?
+states["IA"]
 
 6. With the same hash above, how would we get all the keys?  How about all the values?
+states.keys
+states.values
 
 7. What is another example of when we might use a hash?  In your example, why is a hash better than an array?
+We can store more complex data in a hash, such as a post on instagram. In a hash you can store multiple key value pairs to the index of a particular post. so you could have the values for the images, the caption, the comments etc. stored to the post.
 
-8. What questions do you still have about hashes?
\ No newline at end of file
+8. What questions do you still have about hashes?
+i would like to see more real world examples
```

Comment:
Here's a 'real world' example - you could imagine this existing in a class enrollment system:
```
students = {
  taryn: ["Math", "English", "Computer Science"],
  ellen: ["Computer Science", "English", "Art"]
}
```